card-unification.md

```diff
@@ -0,0 +1,100 @@
+# Card Unification
+
+  * [ ] Change card shape to object with `type`, `name`, `render` and optional `edit` properties
+  * [ ] Card's `type` is validated by renderer (dom renderer cannot render 'text', e.g.)
+  * [ ] Change arguments passed by editor to card's `render` (or `edit`) method
+    * [ ] single argument object with `name`, `onTeardown`, `isInEditor`, `options`, `editorEnv` and `payload` properties
+  * [ ] Return value of card's `render` (or `edit`) method is appended/concatenated by renderer
```

Comment:
:+1: Thanks for the feedback @gdub22. This makes sense, and I'm going to move forward with updating the renderers to work this way (using the return val as the rendered content and handling appending/concatting in the renderer itself)